rustls/src/error.rs

```diff
@@ -533,10 +543,30 @@ impl From<rand::GetRandomFailed> for Error {
     }
 }
 
+/// Any other error that cannot be expressed by other variants in [`Error`].
+///
+/// This can be used by custom crypto provider to expose the underlying error.
+///
+/// Enums holding this type will never compare equal to each other.
```

Comment:
Some more editorial rustdoc tweaks to consider:

```suggestion
/// Any other error that cannot be expressed by a more specific [`Error`] variant.
///
/// For example, an `OtherError` could produced by a custom crypto provider 
/// expose a provider specific error.
///
/// Enums holding this type will never compare equal to each other.
```

Reply:
Good suggestions, just applied them.

Reply:
Sorry, looks like my browser-suggested fixes don't play nicely with `rustfmt` expectations. You'll have to run `cargo rustfmt` over the updated branch to get CI happy again.